ui/StatusQ/src/StatusQ/Controls/StatusTextWithLoadingState.qml

```diff
@@ -0,0 +1,29 @@
+import QtQuick 2.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+StatusBaseText {
+    id: root
+    property bool loading: false
+    property color customColor: Theme.palette.directColor1
+    color: loading ? "transparent" : customColor
+    Loader {
+        anchors.left: parent.left
```

Comment:
It's better to set both anchors here

Reply:
wdym?

Reply:
Also `anchors.right`

Reply:
wont be needed as I specified a width in the loaders source component

Reply:
OK